ui/views.py

```diff
@@ -33,22 +36,25 @@ def get_bundle_url(request, bundle_name):
         return static("bundles/{bundle}".format(bundle=bundle_name))
 
 
-@require_mandatory_urls
-@login_required()
-def dashboard(request, *args):  # pylint: disable=unused-argument
+def _dashboard(request, *args):  # pylint: disable=unused-argument
```

Comment:
and maybe refactoring to a class-based view so that the others can inherit this and add their own changes (but feel free to pospone this big refactoring)